ra2ce/analysis/losses/multi_link_redundancy.py

```diff
@@ -152,27 +152,28 @@ def _is_not_none(value):
                 _current_value = _weighing_analyser.get_current_value()
 
                 if nx.has_path(_graph, u, v):
-                    alt_dist = nx.dijkstra_path_length(
-                        _graph, u, v, weight=self.analysis.weighing.config_value
+                    [_alt_value, _alt_nodes] = nx.single_source_dijkstra(
+                        _graph,
+                        u,
+                        v,
+                        weight=self.analysis.weighing.config_value,
                     )
-                    alt_nodes = nx.dijkstra_path(_graph, u, v)
-                    connected = 1
-                    alt_value = _weighing_analyser.calculate_alternative_value(alt_dist)
+                    _connected = 1
 
-                    diff = round(alt_value - _current_value, 3)
+                    diff = round(_alt_value - _current_value, 3)
                 else:
-                    alt_value = _current_value
-                    alt_nodes, connected = np.NaN, 0
-                    diff = np.NaN
+                    _alt_value = np.nan
+                    _alt_nodes, _connected = np.nan, 0
+                    diff = np.nan
```

Comment:
it makes more sense defining the values beforehand (fallback / default values) then overwriting them in the if.

Reply:
Done in both analyses